std.ddoc

```diff
@@ -37,13 +44,14 @@ function listanchors()
     values.sort();
 
     var newText = "";
-    for(var i = 0; i < values.length; i++) {
+    for (var i in values) {
```

Comment:
[javascript - Why is using "for...in" with array iteration such a bad idea? - Stack Overflow](http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-such-a-bad-idea)